core/parentchain/block-importer/src/block_importer.rs

```diff
@@ -176,11 +190,34 @@ impl<
 				},
 				Err(_) => error!("Error executing relevant extrinsics"),
 			};
-			let queue_game: Option<u64> = self
+			let queue_game: Option<Queue<H256>> = self
 				.ocall_api
 				.get_storage_verified(RegistryStorage::queue_game(), block.header())?
 				.into_tuple()
 				.1;
+			match queue_game {
+				Some(_) => {
+					error!("YEAHHHHH! FOUND QUEUE GAME!!!!!!!!! ");
+					let game_engine = GameEngine::new(1u8, 1u8);
```

Comment:
Can you add a `//FIXME` comment here? (like we hardcode it here, because it's currently being hardcoded in the GameRegistry pallet.